charts/seed-monitoring/charts/prometheus/templates/config.yaml

```diff
@@ -439,7 +439,67 @@ data:
 
     - job_name: vpn-connection
       honor_labels: false
+      metrics_path: /probe
       scrape_interval: 15s
```

Comment:
Agree I think it is also sufficient to scrape them only every 30sec. 
@mvladev What was the reason why we scraped every 15sec in the old vpn-connection job?

Reply:
We'll get a faster indication of a problem.

Reply:
Hmm, but we will sent out an alert only if an issue exists for at least `5m`.
We could detect an issue max `15s` earlier. On the other hand side we need to store the double amount of samples. For my feeling it would be sufficient to scrape every `30s`. What do you think?